notebooks/src/el-paso.ipynb

```diff
@@ -272,6 +272,26 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "def get_historical_data(date: date, column_name):\n",
+    "    \"\"\"\n",
+    "    Look up the value of a parameter on a given date\n",
+    "    in the historical data we've loaded.\n",
+    "    \n",
+    "    Return the value or raise a KeyError if we don't have it.\n",
+    "    \"\"\"\n",
+    "    \n",
+    "    # prefer Texas government data over @KrisMoore compiled data\n",
+    "    try: \n",
+    "        value = el_paso_cases.loc[date, column_name]\n",
+    "        if np.isnan(value):\n",
+    "            raise KeyError\n",
+    "        return value\n",
+    "    except KeyError:\n",
+    "        value = compiled_data.loc[date, column_name]\n",
+    "        if np.isnan(value):\n",
+    "            raise KeyError\n",
+    "        return value\n",
```

Comment:
good point :p

Reply:
Wait actually, this would always throw that exception on NaN, rather than failing over to the compiled_data on NaN if possible

But it's a good idea to have some sort of string with my custom KeyError, will implement that part of it

Reply:
(also BTW we're stuck on Python 3.6.9 because that's what's available in Colab)